models/reviews.models.js

```diff
@@ -33,3 +33,33 @@ exports.updateReviewById = (review_id, inc_votes = "undefined") => {
       return review;
     });
 };
+
+exports.fetchReviews = (category) => {
+  const validCategoryValues = [
+    "euro_game",
+    "social_deduction",
+    "dexterity",
+    "childrens_games",
+  ];
+
+  if (!validCategoryValues.includes(category) && category !== undefined) {
+    return Promise.reject({ status: 400, message: "Invalid category" });
+  }
+
+  let queryStr = `SELECT reviews.*, COALESCE(count_table.comment_count, 0) as comment_count
```

Comment:
OOH coalesce? I need to google this... so it's like count but discounts null values? Nice